packages/components/src/Tooltip/_docs/Tooltip.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react"
+import { Decorator, Meta, StoryFn, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic/isChromatic"
+import { Tag } from "../../Tag"
+import { Button, InformationIcon, Text } from "../../index"
+import { Tooltip } from "../index"
+
+const openTooltipInChromatic: Decorator = (Story, { args }) => {
+  if (isChromatic()) args.isInitiallyVisible = true
+  return <Story />
+}
+
+const meta = {
+  title: "Components/Tooltip",
+  component: Tooltip,
+  args: {
+    text: "Example tooltip text.",
+  },
+  decorators: [openTooltipInChromatic],
+} satisfies Meta<typeof Tooltip>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+  render: args => (
+    <div
+      style={{ marginTop: "100px", display: "flex", justifyContent: "center" }}
+    >
+      <Tooltip {...args}>
+        <Button label="Hover or focus me" />
+      </Tooltip>
+    </div>
+  ),
```

Comment:
Optional clean up: Move the div to a decorator (the code snippet will be a bit cleaner for the consumers)